src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/caseupdated/CaseUpdatedAboutToSubmitHandler.java

```diff
@@ -92,9 +90,8 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         }
 
         if (sscsCaseData.getAppeal().getAppellant() != null
-                && sscsCaseData.getAppeal().getAppellant().getAddress() != null
-                && isNotBlank(sscsCaseData.getAppeal().getAppellant().getAddress().getPostcode())) {
-
+            && sscsCaseData.getAppeal().getAppellant().getAddress() != null
+            && isNotBlank(sscsCaseData.getAppeal().getAppellant().getAddress().getPostcode())) {
```

Comment:
Maybe hold the appellant in a local variable before the if will be easier to read